solutions/community-deploy/cloud/AWS/main.tf

```diff
@@ -154,11 +158,14 @@ resource "aws_instance" "cortx_deploy" {
 }
 
 resource "aws_ebs_volume" "data_vol" {
-  count             = 9
+  count             = var.ebs_volume_count * var.instance_count
   availability_zone = data.aws_availability_zones.available.names[0]
   size              = 10
+  tags = {
```

Comment:
Have you tested with this tag ? as i have also tried similar thing but on the name column was still showing blank for me.

Reply:
Yes. Volumes are getting created with proper names 

![image](https://user-images.githubusercontent.com/63289460/180465267-d0907f5f-d42e-4ed7-83f7-8c425f6343f7.png)